Source/WebCore/Modules/credentialmanagement/DigitalCredentialRequestOptions.idl

```diff
@@ -23,7 +23,7 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-dictionary IdentityRequestOptions {
+dictionary DigitalCredentialRequestOptions {
```

Comment:
Not new to this PR, but shouldn't this have an `EnabledBySetting=`?

Reply:
Not needed since the dictionary type is actually not exposed to JS.

Reply:
Ah, that's just made me realize that we only put `EnabledBySetting=` on the new method for the Digital Credential API. We didn't add it to the resulting interface object 😰.... adding that now.

Reply:
https://github.com/WebKit/WebKit/pull/23289